packages/Angular/explorer-core/src/lib/navigation/navigation.component.ts

```diff
@@ -902,6 +905,24 @@ export class NavigationComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
 
+  protected async loadFiles() {
+    const rv = new RunView();
+    const viewResults = await rv.RunView({ EntityName: 'File Storage Providers' });
```

Comment:
@cadam11 this is good and functional, longer term, we might want to consider creating a Dataset for File Storage metadata and using that instead because that can take advantage of the client-side caching in Datasets